internal/types/staticct/staticct.go

```diff
@@ -166,6 +166,21 @@ type Entry struct {
 	LeafIndex         uint64
 }
 
+// UnmarshalText implements encoding/TextUnmarshaler and reads EntryBundles
+// which are encoded using the Static CT API spec.
+func UnmarshalTimestamp(raw []byte) (uint64, error) {
+	s := cryptobyte.String(raw)
+	var t uint64
+
+	if !s.ReadUint64(&t) || t > math.MaxInt64 {
+		return 0, fmt.Errorf("invalid data tile: timestamp can't be extracted")
+	}
+	if t > math.MaxInt64 {
+		return 0, fmt.Errorf("invalid data tile: timestamp %d > math.MaxInt64", t)
+	}
```

Comment:
Did you mean to duplicate the check here?

Reply:
woops.